### PR TITLE
Clean supported Ruby versions

### PR DESCRIPTION
**Pull request by nicolasleger**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 before_install:
-    - gem install bundler
+  - gem install bundler
 language: ruby
 rvm:
-    - 2.3.0
+  - 1.9.3
+  - 2.0
+  - 2.1
```
Comment (RobertAudi) [inline .travis.yml]: Support for Ruby 2.1 has ended in [April 2017](https://www.ruby-lang.org/en/news/2017/04/01/support-of-ruby-2-1-has-ended/) and we do not intend to support it either.

File: lib/tepee.rb [hunk 2]
```diff
@@ -10,7 +10,8 @@ class << self
 
     protected
 
-    def add(name, default = nil, env_var: "#{env_var_prefix}#{name}")
+    def add(name, default = nil, options = {})
+      env_var = options[:env_var] || "#{env_var_prefix}#{name}"
```
Comment (bankair) [inline lib/tepee.rb]: Hey ! Thanks for that. However, we don't want the add method to accept any kind of input (a typo to `env_var` would go unnoticed).

File: .travis.yml (hunk 1 above)
Comment (nicolasleger) [reply .travis.yml]: Thanks, I removed it and bump the gem spec.

**Comment by nicolasleger** [comment]
Thanks @RobertAudi @bankair 
I cleaned unsupported Ruby versions with related compatibility stuff and specify 2.2.0 requirement.
Better to squash the whole branch while merging.

**Comment by RobertAudi** [comment]
Closed in favor of #5 and #6.

**Pull request closed without merging**